src/packages.builds

```diff
@@ -21,15 +21,6 @@
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
     <!-- add specific builds / pkgproj's here to include in servicing builds -->
-    <Project Include="$(MSBuildThisFileDirectory)..\pkg\Microsoft.Private.CoreFx.NETCoreApp\Microsoft.Private.CoreFx.NETCoreApp.builds">
```

Comment:
I'm debating whether or not we should just always keep this package building. For now go ahead and remove it but at the first change this will need to be added back. What do you think should we always keep it.

Reply:
I was thinking the same thing when I was going to delete it. Any change we take for servicing will cause this to be rebuilt so I think we should keep it. I will add it back and I'll bump the version for this guy then.